interceptors/selector/selector.go

```diff
@@ -0,0 +1,51 @@
+package selector
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+)
+
+type MatchFunc func(ctx context.Context, fullMethod string) bool
+
+// Allow After the method is matched, the interceptor is run
+func Allow(methods []string) MatchFunc {
+	return func(ctx context.Context, fullMethod string) bool {
+		for _, s := range methods {
+			if s == fullMethod {
+				return true
+			}
+		}
+		return false
+	}
+}
+
+// Block the interceptor will not run after the method matches
+func Block(methods []string) MatchFunc {
+	allow := Allow(methods)
+	return func(ctx context.Context, fullMethod string) bool {
+		return !allow(ctx, fullMethod)
+	}
+}
```

Comment:
I don't think we need these, to be honest. Users can easily implement these themselves.